### PR TITLE
Notifications with context

### PR DESCRIPTION
**Pull request by bognix**
## TODO

- [x] send context with newly created questions
- [x] use context to navigate to question/answer
- [x] scroll to element
- [x] create script to tag questions backward
- [x] reactions and context
- [x] highlight sugar (ping @adamkarminski  :D )
- [x] query parameters should persist after redirect
- [x] expand all answers to make sure it's visible
- [x] reload QNA on query param change

## TO CHECK
- [x] navigate to the same page as user is now - is QNA refreshed?

## BUGS
- [x] clear query when question no longer has to be highlighted

### CODE REVIEW

File: resources/assets/js/components/global/form/Form.vue [hunk 1]
```diff
@@ -39,6 +39,7 @@
 			'hideDefaultSubmit',
 			'suppressEnter',
 			'resetAfterSubmit',
+			'meta'
```
Comment (bognix) [inline resources/assets/js/components/global/form/Form.vue]: niepotrzebne

File: resources/assets/js/store/modules/qna.js [hunk 2]
```diff
@@ -198,7 +178,9 @@ const getters = {
 				return sortByVotes(list)
 		}
 	},
-
+	getQuestionIndex: state => (id, list) => {
```
Comment (bognix) [inline resources/assets/js/store/modules/qna.js]: niepotrzebne

File: resources/assets/js/components/global/Dropdown.vue [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+		height: 100%
+		max-width: $navbar-height
+		min-height: 100%
+		min-width: $navbar-height
+		position: relative
+		width: $navbar-height
+
+	.drawer
+		border: $border-light-gray
+		-webkit-box-shadow: 2px 2px 10px 5px rgba(67,73,90,0.3)
```
Comment (bognix) [inline resources/assets/js/components/global/Dropdown.vue]: nasi uzytkownicy są "cutting edge" - nie potrzeba browser prefix - http://caniuse.com/#search=box-shadow

File: resources/assets/js/components/notifications/feed.js [hunk 4]
```diff
@@ -7,26 +7,21 @@ import _ from 'lodash'
...
-	data() {
-		return {
-			hasMore: true,
-		}
-	},
 	computed: {
 		...mapGetters('notifications', [
 			'getOldestNotification',
 			'getSortedNotifications',
+			'hasMore',
```
Comment (bognix) [inline resources/assets/js/components/notifications/feed.js]: niepotrzebne

File: resources/assets/js/components/qna/QnaAnswer.vue [hunk 5]
```diff
@@ -74,6 +74,22 @@
...
 		margin-bottom: $margin-big
 
+		&.isHighlighted
+			@keyframes colorchange
+				0%
+					background: blue
+				100%
+					background: initial
+
+			@-webkit-keyframes colorchange
```
Comment (bognix) [inline resources/assets/js/components/qna/QnaAnswer.vue]: prefix potrzebny przy androidzie < 4.4. Chyba nasi użytkownicy są bardziej "cutting edge"

Comment (adamkarminski) [inline resources/assets/js/components/qna/QnaAnswer.vue]: Cutting edge!

File: resources/assets/js/components/notifications/feed.js (hunk 4 above)
Comment (adamkarminski) [inline resources/assets/js/components/notifications/feed.js]: [Used here](https://github.com/bethinkpl/wnl-platform/blob/notifications-v2/resources/assets/js/components/notifications/feeds/personal/PersonalFeed.vue#L200-L202)

Comment (bognix) [inline resources/assets/js/components/notifications/feed.js]: mixins :D

**Comment by adamkarminski** [comment]
🎉

**Pull request merged**